### ISSUE TITLE
Recurrent tonsillitis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6415]
**Preferred term label:**
Recurrent tonsillitis

**Synonyms**
Recurrent Inflammation of the tonsils

**Definition (free text, please give PubMed ID)**
A frequent inflammation of the tonsils.
PMID:  20948776

**Parent term (use hpo.jax.org/app)**
Tonsillitis

**Comment by pnrobinson** [comment]
The existing term Tonsillitis is actually referring to the Phenotype of a history of recurrent tonsillitis, rather than the disease of acute tonsillitis. I think it is best to adapt the label of this term 
https://hpo.jax.org/app/browse/term/HP:0011110
We would model a single bout of Tonsillitis using the corresponding MONDO term
